packages/components/table/src/table-body/index.ts

```diff
@@ -28,15 +28,52 @@ export default defineComponent({
       useRender(props)
     const { onColumnsChange, onScrollableChange } = useLayoutObserver(parent!)
 
+    const hoveredCellList = []
     watch(props.store.states.hoverRow, (newVal: any, oldVal: any) => {
+      const el = instance?.vnode.el as HTMLElement
+      const rows = Array.from(el?.children || []).filter((e) =>
+        e?.classList.contains(`${ns.e('row')}`)
+      )
+
+      // hover rowSpan > 1 choose the whole row
+      const childNodes = rows[newVal]?.childNodes
+      if (childNodes?.length) {
+        const indexes = Array.from(childNodes).reduce((acc, item, index) => {
+          // drop colsSpan
+          const pre = childNodes[index - 1]?.colSpan > 1
+          const next = childNodes[index + 1]?.colSpan > 1
```

Comment:
I'm a little confused, why is `colSpan` being judged here? 🤔

Reply:
I tested your code and it works satisfactorily. But I have some questions about the code, can you explain it? @dowinweb

Reply:
yeah sorry i forget submit the comments , it was pending last week🤣